data/transactions/logic/opcodes.go

```diff
@@ -81,6 +81,7 @@ const spliceVersion = 10  // box splicing/resizing
 const incentiveVersion = 11 // block fields, heartbeat
 
 const spOpcodesVersion = 11 // falcon_verify, sumhash512
+const bmodexpVersion = 11   // bmodexp
```

Comment:
I think we can dispense with the elaborate version constants and "experimental" stuff for this opcode. When we merge it, we're committing to get it into the next version.